.changeset/changelog-stylelint.js

```diff
@@ -5,9 +5,44 @@
 const { getInfo, getInfoFromPullRequest } = require('@changesets/get-github-info');
 
 /**
- * @type {import('@changesets/types').ChangelogFunctions}
+ * @type {Array<string | undefined>}
+ */
+const CHANGESET_SECTIONS = ['Deprecated', 'Removed', 'Changed', 'Added', 'Fixed'];
+
+const changesetSectionReg = new RegExp(`^(${CHANGESET_SECTIONS.join('|')}): `);
+
+/**
+ * @typedef { 'major' | 'minor' | 'patch' } ReleaseType
+ * @typedef { Record<ReleaseType, Array<Promise<string>>> } ReleaseLines
+ * @typedef { Record<ReleaseType, string[]> | string[] } ResolvedReleaseLines
+ * @type {import('@changesets/types').ChangelogFunctions & { reorderReleaseLines(releaseLines: ReleaseLines): Promise<ResolvedReleaseLines> }}
  */
 const changelogFunctions = {
+	async reorderReleaseLines(releaseLines) {
+		const resolved = {
+			major: await Promise.all(releaseLines.major),
+			minor: await Promise.all(releaseLines.minor),
+			patch: await Promise.all(releaseLines.patch),
+		};
+
+		return Object.entries(resolved)
+			.reduce((acc, [_type, lines]) => {
+				const type = /**  @type {ReleaseType} */ (_type);
+				lines.forEach((line) => {
+					if (line) {
+						acc.push(`${line} - ${type}${type === 'major' ? ' (BREAKING)' : ''}`);
```

Comment:
[suggest] I noticed an extra suffix `- minor` at the end of the line below when running `npm run version`:

```diff
--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,5 +1,9 @@
 # Changelog
 
+## 14.12.0
+
+- Added: `declaration-block-no-duplicate-properties` autofix ([#6296](https://github.com/stylelint/stylelint/pull/6296)) ([@fpetrakov](https://github.com/fpetrakov)). - minor
+
 ## 14.11.0
 
 - Added: `ignoreAfterCombinators: []` to `selector-max-universal` ([#6275](https://github.com/stylelint/stylelint/pull/6275)).
```

Can the following change fix it?

```suggestion
						acc.push(`${line}${type === 'major' ? ' (BREAKING)' : ''}`);
```

Reply:
So I misunderstood your comment before. 🤣